pwdtk/helpers.py

```diff
@@ -72,3 +21,28 @@ def recursion_depth():
         if path == top_frame[0] and func_name == top_frame[2]:
             counter += 1
     return counter
+
+
+def _resolve_object_path(dotted_name):
+    if isinstance(dotted_name, six.string_types):
```

Comment:
isn't there something already working, that's doing the same??

Reply:
i copy this function from https://github.com/adamchainz/django-jsonfield/blob/master/jsonfield/utils.py

Reply:
why did you prefer the other one?

Reply:
because the author of django-jsonfield  does something manually (with a loop) which is already done by a standard default python module.

The author of django-jsonfield wrote probably his code to be compatible with python 2.6
From python 2.7 there is a built in importlib, so  better to use this one.

I don't think we want to be python 2.6 compatible any more ;-)